knowledge_repo/app/templates/post_editor_markdown.html

```diff
@@ -0,0 +1,234 @@
+{% extends "post_editor_base.html" %}
```

Comment:
just for my own understanding, what you did with the `html` files is to break it out from 1 template into 4 different ones yes?